Recursive/MultiDimensionalArrayPrint.js

```diff
@@ -0,0 +1,18 @@
+function printElements(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    if (Array.isArray(arr[i])) {
+      // Recursive case: if the element is an array, go deeper
+      printElements(arr[i])
+    } else {
+      // Base case: if the element is not an array, print it
+      console.log(arr[i])
+    }
+  }
+}
```

Comment:
Always use `return` in the function and console the function execution .

Reply:
It is the function used to print the element, why should I use "Return"?